sympy/core/tests/test_var.py

```diff
@@ -1,83 +1,87 @@
 # Tests for var are in their own file, because var pollutes global namespace.
-
 from sympy import Symbol, var, Function, FunctionClass
 from sympy.utilities.pytest import raises
 
-# make z1 with call-depth = 1
+def test_var():
+    exec(("""
+var("a")
+assert a == Symbol("a")
+
+var("b bb cc zz _x")
+assert b == Symbol("b")
+assert bb == Symbol("bb")
+assert cc == Symbol("cc")
+assert zz == Symbol("zz")
+assert _x == Symbol("_x")
+
+v = var(['d', 'e', 'fg'])
+assert d == Symbol('d')
+assert e == Symbol('e')
+assert fg == Symbol('fg')
 
+# check return value
+assert v == [d, e, fg]
+
+# make z1 with call-depth = 1
 
 def _make_z1():
-    var("z1")
+    z1 = var("z1")
```

Comment:
Why have you added `z1 = var("z1")`? If I'm not mistaken, the whole point was that call `var("z1")` itself creates a variable =/ The same applies to `z2 = var("z2")`

Reply:
Done the required changes and committed them.